src/utils/hooks/useQuickStarts/useQuickStarts.ts

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  getBestMatch,
+  groupQuickStartsByName,
+} from '@kubevirt-utils/hooks/useQuickStarts/utils/utils';
+import { QuickStartModel } from '@kubevirt-utils/models';
+import {
+  getGroupVersionKindForModel,
+  useK8sWatchResource,
+  WatchK8sResult,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { getDisabledQuickStarts, QuickStart } from '@patternfly/quickstarts';
+
+const useQuickStarts = (filterDisabledQuickStarts = true): WatchK8sResult<QuickStart[]> => {
+  const preferredLanguage = useTranslation().i18n.language;
```

Comment:
its nicer to deconstruct on left side then the right

Reply:
I'm not sure what you mean. Could you explain a bit more and explain why?